lib/src/rendering/gap.dart

```diff
@@ -3,54 +3,54 @@ import 'package:flutter/rendering.dart';
 
 class RenderGap extends RenderBox {
   RenderGap({
-    double mainAxisExtent,
-    double crossAxisExtent,
-    Axis fallbackDirection,
-    Color color,
+    double? mainAxisExtent,
+    double? crossAxisExtent,
```

Comment:
It cannot be null. Because we set this parameter to 0 if it's null in the widget